crates/gpui/src/key_dispatch.rs

```diff
@@ -75,6 +75,7 @@ pub(crate) struct DispatchTree {
     view_node_ids: FxHashMap<EntityId, DispatchNodeId>,
     keymap: Rc<RefCell<Keymap>>,
     action_registry: Rc<ActionRegistry>,
+    modifiers_changed_listeners: Vec<DispatchNodeId>,
```

Comment:
I don't think this change is needed at all, modifiers changed events are already global.

Reply:
That's odd...when I get rid of `modifiers_changed_listeners`, the events are no longer global. I think a modifier event set directly on the window is global, but not one set through the DispatchTree?

Reply:
hmmm....

Reply:
I am definitely incorrect that modifiers changed events are global. In fact, none of the ones I thought where global currently are....